tests/reporting/test_fee_distribution.py

```diff
@@ -349,6 +349,56 @@ def test_forking(localFixture, universe, market, categoricalMarket, cash, reputa
             with TokenDelta(newReputationToken, expectedRep, account, "Redeeming didn't increase REP correctly"):
                 reportingParticipant.redeem(account, startgas=long(6.7 * 10**6))
 
+def test_forkAndRedeem(localFixture, universe, market, categoricalMarket, cash, reputationToken):
+    # Let's do some initial disputes for the categorical market
+    proceedToNextRound(localFixture, categoricalMarket, tester.k1, moveTimeForward = False)
+
+    # Get to a fork
+    testers = [tester.k0, tester.k1, tester.k2, tester.k3]
+    testerIndex = 1
+    while (market.getForkingMarket() == longToHexString(0)):
+        proceedToNextRound(localFixture, market, testers[testerIndex], True)
+        testerIndex += 1
+        testerIndex = testerIndex % len(testers)
+
+    # Have the participants fork and create new child universes
+    for i in range(market.getNumParticipants()):
+        reportingParticipant = localFixture.applySignature("DisputeCrowdsourcer", market.getReportingParticipant(i))
```

Comment:
I think this loop can be removed. `reportingParticipant` doesn't seem to be getting used for anything until the for loop below.

Reply:
ah yea good catch